src/olav_espenes_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,21 @@
+def bubble_sort(file):
+    """Sort list with number in increasing order
+    """
+    data_list = list(file)
+    length_list = len(file)
+
+    for element in range(length_list):
+        for sec_element in range(0, length_list - element - 1):
```

Comment:
`range` starts from 0, so `for sec_element in range(length_list - element - 1):` is enough